### ISSUE TITLE
add function to copy all function name

### ISSUE THREAD

**Issue by FFengIll** [patrys/vscode-code-outline#97]
For convenience , I wish to add a button (or something else) to copy the symbol list from panel node.

May I do a PR for it?

**Comment by KamasamaK** [comment]
I don't believe there is anything in the VS Code API for accessing the clipboard, but you should be able to use a Node package like [`clipboardy`](https://www.npmjs.com/package/clipboardy) to accomplish that.
